fili-generic-example/README.md

```diff
@@ -13,6 +13,15 @@ Note that this was last tested using [version 0.9.1](https://github.com/yahoo/fi
 
 ## Setup and Launching
 
+### Docker
+There is a Fili Dockerfile which is available on Dockerhub. To start playing around with Fili, you can download and open Docker. Then
+run the command
+```bash
+docker run --name fili-generic-example -i --rm -p 3001:8081 -p 3000:8082 fili:1.0
```

Comment:
These were the commands I had to run to start it up

```bash
docker pull mpardesh/fili
docker run --name fili-generic-example -i --rm -p 3001:8081 -p 3000:8082 mpardesh/fili:1.0
```
also there should probably be a `-p 9998:9998` so that people can connect to fili.